keyboards/enviousdesign/60f/info.json

```
@@ -0,0 +1,227 @@
+                {"label": "0,0", "matrix": [0, 0], "x": 2.5, "y": 0},
+                {"label": "0,1", "matrix": [0, 1], "x": 3.5, "y": 0},
+                {"label": "0,2", "matrix": [0, 2], "x": 4.5, "y": 0},
+                {"label": "0,3", "matrix": [0, 3], "x": 5.5, "y": 0},
+                {"label": "0,4", "matrix": [0, 4], "x": 6.5, "y": 0},
+                {"label": "0,5", "matrix": [0, 5], "x": 7.5, "y": 0},
+                {"label": "0,6", "matrix": [0, 6], "x": 8.5, "y": 0},
+                {"label": "0,7", "matrix": [0, 7], "x": 9.5, "y": 0},
+                {"label": "0,8", "matrix": [0, 8], "x": 10.5, "y": 0},
+                {"label": "0,9", "matrix": [0, 9], "x": 11.5, "y": 0},
+                {"label": "0,10", "matrix": [0, 10], "x": 12.5, "y": 0},
+                {"label": "0,11", "matrix": [0, 11], "x": 13.5, "y": 0},
+                {"label": "0,12", "matrix": [0, 12], "x": 14.5, "y": 0},
+                {"label": "0,14", "matrix": [0, 14], "x": 15.5, "y": 0, "w": 2},
+                {"label": "1,0", "matrix": [1, 0], "x": 2.5, "y": 1, "w": 1.5},
+                {"label": "1,2", "matrix": [1, 2], "x": 4, "y": 1},
+                {"label": "1,3", "matrix": [1, 3], "x": 5, "y": 1},
+                {"label": "1,4", "matrix": [1, 4], "x": 6, "y": 1},
+                {"label": "1,5", "matrix": [1, 5], "x": 7, "y": 1},
+                {"label": "1,6", "matrix": [1, 6], "x": 8, "y": 1},
+                {"label": "1,7", "matrix": [1, 7], "x": 9, "y": 1},
+                {"label": "1,8", "matrix": [1, 8], "x": 10, "y": 1},
+                {"label": "1,9", "matrix": [1, 9], "x": 11, "y": 1},
+                {"label": "1,10", "matrix": [1, 10], "x": 12, "y": 1},
+                {"label": "1,11", "matrix": [1, 11], "x": 13, "y": 1},
+                {"label": "1,12", "matrix": [1, 12], "x": 14, "y": 1},
+                {"label": "1,13", "matrix": [1, 13], "x": 15, "y": 1},
+                {"label": "Enter", "matrix": [2, 14], "x": 16.25, "y": 1, "w": 1.25, "h": 2},
+                {"label": "2,0", "matrix": [2, 0], "x": 2.5, "y": 2, "w": 1.75},
+                {"label": "2,2", "matrix": [2, 2], "x": 4.25, "y": 2},
+                {"label": "2,3", "matrix": [2, 3], "x": 5.25, "y": 2},
+                {"label": "2,4", "matrix": [2, 4], "x": 6.25, "y": 2},
+                {"label": "2,5", "matrix": [2, 5], "x": 7.25, "y": 2},
+                {"label": "2,6", "matrix": [2, 6], "x": 8.25, "y": 2},
+                {"label": "2,7", "matrix": [2, 7], "x": 9.25, "y": 2},
+                {"label": "2,8", "matrix": [2, 8], "x": 10.25, "y": 2},
+                {"label": "2,9", "matrix": [2, 9], "x": 11.25, "y": 2},
+                {"label": "2,10", "matrix": [2, 10], "x": 12.25, "y": 2},
+                {"label": "2,11", "matrix": [2, 11], "x": 13.25, "y": 2},
+                {"label": "2,12", "matrix": [2, 12], "x": 14.25, "y": 2},
+                {"label": "2,13", "matrix": [2, 13], "x": 15.25, "y": 2},
```

Enter needs to be on the Home Row for `60_iso` Community Layout compatibility.

```suggestion
                {"label": "2,0", "matrix": [2, 0], "x": 2.5, "y": 2, "w": 1.75},
                {"label": "2,2", "matrix": [2, 2], "x": 4.25, "y": 2},
                {"label": "2,3", "matrix": [2, 3], "x": 5.25, "y": 2},
                {"label": "2,4", "matrix": [2, 4], "x": 6.25, "y": 2},
                {"label": "2,5", "matrix": [2, 5], "x": 7.25, "y": 2},
                {"label": "2,6", "matrix": [2, 6], "x": 8.25, "y": 2},
                {"label": "2,7", "matrix": [2, 7], "x": 9.25, "y": 2},
                {"label": "2,8", "matrix": [2, 8], "x": 10.25, "y": 2},
                {"label": "2,9", "matrix": [2, 9], "x": 11.25, "y": 2},
                {"label": "2,10", "matrix": [2, 10], "x": 12.25, "y": 2},
                {"label": "2,11", "matrix": [2, 11], "x": 13.25, "y": 2},
                {"label": "2,12", "matrix": [2, 12], "x": 14.25, "y": 2},
                {"label": "2,13", "matrix": [2, 13], "x": 15.25, "y": 2},
                {"label": "Enter", "matrix": [2, 14], "x": 16.25, "y": 1, "w": 1.25, "h": 2},
```

---

Could this be addressed?

---

The enter key on this keyboard is shifted a row to make space for ansi backslash, does this mean its incompatible with community layout and should be removed from the info.json?